components/SendCardProfileSearch.vue

```diff
@@ -74,34 +69,65 @@ const handleReceiverSearch = async (event: CustomEvent) => {
   isSearchingReceiver.value = false
 }
 
-const handleSelect = async (event: CustomEvent) => {
-  const selection = event.detail.value as SearchProfileResult
-  const { address, name, image } = selection
-  searchTerm.value = address
+const selectProfile = async (address?: Address) => {
+  const searchResults = await searchProfile(address)
+
+  if (!searchResults || searchResults.hits.length === 0) {
+    return
+  }
+
+  const [selectedProfile] = searchResults.hits.map(hit => {
+    return {
+      name: hit.LSP3Profile?.name,
+      address: hit.address,
+      image: hit.profileImageUrl,
+    }
+  })
+
   receiver.value = {
     address,
-    name,
+    name: selectedProfile.name,
     profileImage: [
       {
-        src: image,
+        src: selectedProfile.image,
       },
     ],
   }
   receiverError.value = ''
   results.value = undefined
 }
 
-const handleBlur = () => {
-  const { formatMessage } = useIntl()
+const searchProfile = async (searchTerm?: string) => {
+  if (!searchTerm) {
+    return
+  }
 
+  return await search({
+    query: searchTerm,
+    requestOptions: {
+      hitsPerPage: SEARCH_RESULTS_LIMIT,
+      page: 0,
+    },
+  })
+}
+
+const handleSelect = async (event: CustomEvent) => {
+  const selection = event.detail.value as SearchProfileResult
+  const { address } = selection
+  await selectProfile(address)
+  searchTerm.value = address
+}
+
+const handleBlur = () => {
   // we add slight delay to allow `on-select` to be triggered first
-  setTimeout(() => {
+  setTimeout(async () => {
     if (searchTerm.value && !isAddress(searchTerm.value)) {
       receiverError.value = formatMessage('errors_invalid_address')
     } else {
       receiverError.value = ''
+      await selectProfile(searchTerm.value as Address)
     }
-  }, 300)
+  }, 100)
```

Comment:
Maybe create some const var to easily spot on this. Only in this file would be fine